inspirehep/dojson/hep/model.py

```diff
@@ -26,4 +26,19 @@
 from dojson import utils
 
 hep = Overdo()
+
+
+@hep.over('book', '^773..')
```

Comment:
Good point @jalavik Actually did not read this comment in time and merged https://github.com/inspirehep/inspire-next/pull/666 after a slight modification that allows to keep using `hep.do`

The effect is the same, so at least somebody can learn how to inject a function into a class instance :)